quarkus/dist/src/main/content/bin/kc.sh

```diff
@@ -1,5 +1,32 @@
 #!/bin/bash
 
+# With added utility function.
+function printArrayElementsInLines() {
+    local sourceArray=${!1}
+    printf "%s\n" ${sourceArray[@]}
+}
+
+function checkIfArrayContains() {
+    local sourceArray=${!1}
+    local stringToSearch="$2"
+    printArrayElementsInLines sourceArray[@] | grep -q "^$stringToSearch$"
+}
+
+function countOccurrencesInArray() {
+    local sourceArray=${!1}
+    local stringToSearch="$2"
+    let resultOccurrencesCount=0
+
+    for oneElement in ${sourceArray[*]}; do
+        if [ $oneElement == $stringToSearch ]; then
+            resultOccurrencesCount=$((resultOccurrencesCount + 1))
+        fi
+    done
+
+    return ${resultOccurrencesCount}
```

Comment:
`return` has awful semantics in bash, we can use it in a more "functional" way by `echo`ing and assigning directly the result.

Reply:
Thanks for your comments and suggestions.

I've tested changing from `return ${resultOccurrencesCount}` to `echo ${resultOccurrencesCount}` and I didn't reach the desired outcome. Then I kept the "return" reserved word on that function.